examples/docs/en-US/dropdown.md

```diff
@@ -126,6 +129,27 @@ Use `hide-on-click` to define if menu closes on clicking.
 ```
 :::
 
+### Command event
+
+点击菜单项后会触发事件，用户可以通过相应的菜单项 key 进行不同的操作
```

Comment:
Clicking each dropdown item fires an event whose parameter is assigned by each item.